list.json

```
@@ -162,4 +162,5 @@
   "uxkjaer/featureswitch",
   "xamafe/RFC_TOOLS",
   "SAP/project-odense"
```

thanks, add a comma here to fix the JSON syntax,

```suggestion
  "SAP/project-odense",
```